cypress/support/testHelper.js

```diff
@@ -17,3 +17,25 @@ export const shouldContainStyles = (element, css, styling) => {
     expect(el).to.have.css(css, styling);
   });
 };
+
+export const checkElementStyles = (elementString, text, color, fontFamily) => {
+  const el = getElement(elementString);
+  shouldContainText(el, text);
+  shouldContainStyles(el, 'color', color);
+  shouldContainStyles(el, 'font-family', fontFamily);
+};
+
+export const figureVisibility = figure => {
```

Comment:
These 3 utils are very specific and are not really re-used. It would be simpler to not prematurely optimise, and just optimise when/if we move image tests to another file.

Reply:
We created these due to CodeClimate's request for refactoring duplication. The initial iteration of this just included the simple statements.

Reply:
I would be happy not being so dogmatic with CodeClimate here, if we are aware that we may refactor it in the Image milestone anyway. Thoughts?